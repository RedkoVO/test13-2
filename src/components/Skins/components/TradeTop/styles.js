import WhiteDown from '../../../../assets/images/svg/white-down.svg'
import CheckedIcon from '../../../../assets/images/svg/checked.svg'

export default () => ({
  root: {
    display: 'flex',
    flexDirection: 'row',
    alignItems: 'center',
    justifyContent: 'space-around',
    minWidth: '800px',
    padding: '0 20px',
    height: '80px',
    backgroundColor: 'rgba(0,0,0,0.08)',
    borderRadius: '15px'
  },
  tradeGame: {
    display: 'flex',
    flexDirection: 'row',
    alignItems: 'center',
    justifyContent: 'center',
    minWidth: '200px'
  },
  tradeGameSelect: {
    fontSize: '16px',
    fontWeight: 300,
    color: '#fff',
    marginRight: '10px'
  },
  tradeGameItem: {
    fontSize: '30px',
    fontWeight: 700,
    color: '#fff',
    position: 'relative',
    paddingRight: '25px',
    textDecoration: 'none',

    '&:after': {
      position: 'absolute',
      content: `''`,
      height: '8px',
      width: '15px',
      background: `url(${WhiteDown})`,
      backgroundRepeat: 'no-repeat',
      backgroundPosition: 'center',
      backgroundSize: 'cover',
      right: '5px',
      top: '50%',
      transform: 'translateY(-50%)'
    }
  },
  tradeStream: {
    maxWidth: '550px',
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'flex-start',
    justifyContent: 'center'
  },
  tradeStreamTitle: {
    color: '#fff',
    fontSize: '16px',
    fontWeight: 300,
    paddingLeft: '30px'
  },
  tradeStreamUrlWrap: {
    position: 'relative',
    minWidth: '50%',

    '&:after': {
      position: 'absolute',
      content: `''`,
      height: '9px',
      width: '13px',
      background: `url(${CheckedIcon})`,
      backgroundRepeat: 'no-repeat',
      backgroundPosition: 'center',
      backgroundSize: '13px',
      right: '10px',
      top: '50%',
      zIndex: 100
    }
  },
  tradeStreamUrl: {
    position: 'relative',
    width: '520px',
    maxWidth: 'inherit',
    border: 'none',
    outline: 'none',
    backgroundColor: 'rgba(0,0,0,0.13)',
    padding: '8px 23px',
    boxSizing: 'border-box',
    fontSize: '10px',
    marginTop: '8px',
    borderRadius: '15px'
  }
})
