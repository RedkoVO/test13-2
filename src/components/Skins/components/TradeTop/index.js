import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import styles from './styles'

const TradeTop = ({ classes }) => (
  <div className={classes.root}>
    <div className={classes.tradeGame}>
      <div className={classes.tradeGameSelect}>Game:</div>
      <div className={classes.tradeGameItem}>
        CS:GO
      </div>
    </div>
    <div className={classes.tradeStream}>
      <div className={classes.tradeStreamTitle}>Put your steam trade url: </div>
      <div className={classes.tradeStreamUrlWrap}>
        <input
          type="text"
          className={classes.tradeStreamUrl}
          placeholder="https://steamcommunity.com/groups/dota2lounge/discussions/0/558"
        />
      </div>
    </div>
  </div>
)

TradeTop.propTypes = {
  classes: PropTypes.object
}

export default withStyles(styles)(TradeTop)
