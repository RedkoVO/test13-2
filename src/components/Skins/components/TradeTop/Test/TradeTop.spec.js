import * as React from 'react'
import renderer from 'react-test-renderer'

import TradeTop from '../'

it('renders correctly', () => {
  const tree = renderer.create(<TradeTop />).toJSON()
  expect(tree).toMatchSnapshot()
})
