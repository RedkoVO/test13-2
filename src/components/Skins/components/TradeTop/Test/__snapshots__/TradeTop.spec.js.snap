// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="TradeTop-root-1"
>
  <div
    className="TradeTop-tradeGame-2"
  >
    <div
      className="TradeTop-tradeGameSelect-3"
    >
      Game:
    </div>
    <div
      className="TradeTop-tradeGameItem-4"
    >
      CS:GO
    </div>
  </div>
  <div
    className="TradeTop-tradeStream-5"
  >
    <div
      className="TradeTop-tradeStreamTitle-6"
    >
      Put your steam trade url: 
    </div>
    <div
      className="TradeTop-tradeStreamUrlWrap-7"
    >
      <input
        className="TradeTop-tradeStreamUrl-8"
        placeholder="https://steamcommunity.com/groups/dota2lounge/discussions/0/558"
        type="text"
      />
    </div>
  </div>
</div>
`;
